docs/source/start-routingspec.rst

```diff
@@ -662,22 +717,200 @@ so consider the name before saving.
-|Destination              |consumer (pipe, database, file, etc)               |
-+-------------------------+---------------------------------------------------+
-|Output Format            |set output format                                  |
-+-------------------------+---------------------------------------------------+
-|Date/Time - Since/Util   |define time window for message retreival           |
-+-------------------------+---------------------------------------------------+
-|Platform Selection       |Specify a platform or network list                 |
-+-------------------------+---------------------------------------------------+
++-------------------------+------------------------------------------------+
+|**Field**                |**Description**                                 |
++=========================+================================================+
+|Data Source              |Location in Database                            |
++-------------------------+------------------------------------------------+
+|Destination              |consumer (pipe, database, file, etc)            |
++-------------------------+------------------------------------------------+
+|Output Format            |set output format                               |
++-------------------------+------------------------------------------------+
+|Date/Time - Since/Util   |define time window for message retrieval        |
++-------------------------+------------------------------------------------+
+|Platform Selection       |Specify a platform or network list              |
++-------------------------+------------------------------------------------+
+
+The **Data Source** will populate with a drop down menu consisting of the
+ source or sources added in step 3.  The **Destination** will populate with 
+ some options including but not limited to the database, file and pipe.  
+ To set up a routing spec to go into the database, select the applicable 
+ database.  The **Date/Time** format is only applicable for sources
+ that are not static.  For example, sources of type lrgs or abstract web
+ will require a look back time window to be defined.  The longer the look 
+ back the longer the routing spec will take to run, because there will
+ presumably be more data.  It is good practice to take into consideration
+ how frequently the source is updated, the resolution of the data, 
+ how frequently you are planning to run the routing spec, and how many
+ platforms the routing spec will include.  The **Platform Selection** 
+ is where the user will need to either specify a platform or network list.
+ Recall that a network list is a group of platforms.  
 
 .. image:: ./media/start/routingspec/im-28-rs-source.JPG
    :alt: new list window
    :width: 500
 
+NOTE - there are some unique criteria that need to be selected given 
+certain types of sources.  For example, when the source type is a 
+directory, then the property "oneScanOnly" must be defined as True.
```

Comment:
```suggestion
directory, then the property "oneScanOnly" can be defined as True 
to prevent continuously scanning the directory.
```

Reply:
At least in the version LRE has running - this HAS to be set as True.  Maybe perhaps better said if it is not set as True, the routing spec will NEVER close and there will be never ending scanning of the directory.